docs/01_nodeos/03_plugins/trace_api_plugin/index.md

```diff
@@ -101,23 +112,74 @@ nodeos --data-dir data_dir --config-dir config_dir --trace-dir traces_dir
 --trace-rpc-abi=eosio.wrap=abis/eosio.wrap.abi
 ```
 
-## Maintenance Note
+## Definitions
 
-To reduce the disk space consumed by the `trace_api_plugin`, configure the following option: 
+This section provides an overview of *slices*, the *trace log* contents, and the *clog format*. Mastery of these concepts is beneficial for an effective use of the `trace_api_plugin` options.
 
-```console
+### Slices
+
+In the context of the `trace_api_plugin`, a *slice* is defined as a collection of all relevant trace data between a given starting block height (inclusive) and a given ending block height (exclusive). For instance, a slice from 0 to 10,000 is a collection of all blocks with block numbers greater than or equal to 0 and less than 10,000. The trace directory contains a collection of slices. Each slice consists of a *trace data* log file and a *trace index* metadata log file:
+
+  *  `trace_<S>-<E>.log`
+  *  `trace_index_<S>-<E>.log`
+
+where `<S>` and `<E>` are the starting and ending block numbers for the slice padded with leading 0's to a stride. For instance if the start block is 5, the last is 15, and the stride is 4, then the resulting `<S>` is `0000000005` and `<E>` is `0000000015`.
+
+#### trace_&lt;S&gt;-&lt;E&gt;.log
+
+The trace data log is an append only log that stores the actual binary serialized block data. The contents include the transaction and action trace data needed to service the RPC requests augmented by the per-action ABIs. The trace log may include blocks that have been forked out of the blockchain as part of the normal operations of the chain. The next entry in the file will always have a block number one higher than the previous one or the same number or less because of forking.  Every trace entry will have a corresponding entry in the corresponding slice file for trace indexes. Note that forked blocks can be avoided by running nodeos in `read-mode=irreversible`. The log begins with a basic header that includes versioning information about the data stored in the log.
+
+#### trace_index&#95;&lt;S&gt;-&lt;E&gt;.log
+
+The trace index log or metadata log is an append only log that stores a sequence of binary-serialized types. Currently two types are supported: `block_entry_v0` and `lib_entry_v0`. The log begins with a basic header that includes versioning information about the data stored in the log. `block_entry_v0` includes a block ID and block number with an offset to the location of that block within the data log. `lib_entry_v0` includes an entry for the latest known LIB. The reader will use the LIB information for reporting to users an irreversible status.
```

Comment:
By the time of this writing there are actually 3 types of entries, those two and `block_trace_v1` which contains the same data as `block_trace_v0` and **adds** the information from #9005

Reply:
These are the entries for the trace index log file.

Reply:
yes, correct, my mistake